cloud/amazon/ec2.py

```diff
@@ -626,8 +626,18 @@ def get_instance_info(inst):
         instance_info['ebs_optimized'] = getattr(inst, 'ebs_optimized')
     except AttributeError:
         instance_info['ebs_optimized'] = False
+    
+    try:
+        instance_info['vpc_id'] = getattr(inst, 'vpc_id')
+    except AttributeError:
+        instance_info['vpc_id'] = None
```

Comment:
I've used the same strategy from others attributes to keep consistency. 
I can change it and submit again if you prefer so.

On my own code, I'm also returning the instance_profile_name (IAM Role), you're welcome to include this as well:
```python
    try:
        instance_info['instance_profile'] = getattr(inst, 'instance_profile')
        if instance_info['instance_profile']:
            instance_info['instance_profile_name'] = instance_info['instance_profile']['arn'].rsplit('/',1)[-1]
        else:
            instance_info['instance_profile_name'] = None
    except AttributeError:
        instance_info['instance_profile'] = None
```